### PR TITLE
Remove monitoring v1

### PR DESCRIPTION
**Pull request by diogoasouza**
Monitoring v1 was deprecated on rancher 2.5, this PR removes the codebase.

https://github.com/rancher/rancher/issues/43325

### CODE REVIEW

**Review by rohitsakala — CHANGES_REQUESTED** [review]

**Review by vardhaman22 — APPROVED** [review]

**Review by joshmeranda — APPROVED** [review]
LGTM. I was never involved with the monitoring v1 development, so if we can get @aiyengar2 to sign off as well we'll be in good shape here.

File: pkg/monitoring/monitoring.go
[the remark this answers is not in the document]
Comment (joshmeranda) [reply pkg/monitoring/monitoring.go:1]: I think we need some of this code to continue to expose some data from the rancher api to be consumed by users via UI

**Review by aiyengar2 — APPROVED** [review]

File: pkg/data/management/role_data.go [hunk 1]
```diff
@@ -425,12 +420,6 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 	rb.addRoleTemplate("View Project Catalogs", "projectcatalogs-view", "project", false, false, false).
 		addRule().apiGroups("management.cattle.io").resources("projectcatalogs").verbs("get", "list", "watch")
 
-	rb.addRoleTemplate("Project Monitoring View Role", "project-monitoring-readonly", "project", false, true, false).
-		addRule().apiGroups("monitoring.cattle.io").resources("prometheus").verbs("view").
-		setRoleTemplateNames("view")
-
-	rb.addRoleTemplate("View Monitoring", "monitoring-ui-view", "project", true, false, false)
```
Comment (aiyengar2) [inline pkg/data/management/role_data.go:432]: This shouldn’t be removed as it is tied to Monitoring V2

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
missed one issue

**Comment by rohitsakala** [comment]
Closing in favour of https://github.com/rancher/rancher/pull/44229

**Pull request closed without merging**